### PR TITLE
Add deployment script to be used in admin interface

### PR DESCRIPTION
**Pull request by 4p00rv**

### CODE REVIEW

File: js/deploy_contracts.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ */
+function deploy(tokenOptions, deploymentOptions) {
+    deploymentOptions = Object.assign(deploymentDefaults, deploymentOptions);
+    tokenOptions = Object.assign(tokenDefaults, tokenOptions);
+
+    // If testing locally please start the testrpc first.
+    // For Mist enabled browsers Web3.givenProvider will point to the current Ether network in use.
+    let web3 = new Web3(Web3.givenProvider || deploymentOptions.testnet);
+
+    return new Promise((resolve, reject) => {
```
Comment (aminmarashi) [inline js/deploy_contracts.js]: This could be a good place for using `async/await`.

File: test/deployment_script.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        });
+        // Listen on localhost:8546
+        testrpc.listen(8546, '127.0.0.1');
+
+        web3 = new Web3('http://localhost:8546');
+        accounts = await web3.eth.getAccounts();
+        deploymentOptions.testnet = 'http://127.0.0.1:8546';
+    });
+
+    after((done) => {
```
Comment (aminmarashi) [inline test/deployment_script.js]: done is unnecessary.

File: test/deployment_script.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        testrpc.close();
+        done();
+    });
+
+    it('Deploys contract to test net', async () => {
+        [tokenFactoryAddress, tokenAddress] = await deployToken(tokenOptions, deploymentOptions);
+        assert.isDefined(tokenFactoryAddress);
```
Comment (aminmarashi) [inline test/deployment_script.js]: [`Chai expect/should`](http://chaijs.com/api/bdd/) usually results in a more readable test.

Comment (4p00rv) [reply test/deployment_script.js]: @aminmarashi Since this is a fork, and `assert` is used widely, I'm using `assert` for consistency purposes.

**Pull request merged**